packages/grid/_modules_/grid/utils/exportAs.ts

```diff
@@ -12,7 +12,7 @@ import { GridExportFormat } from '../models/gridExport';
  */
 export function exportAs(
   blob: Blob,
-  extension: GridExportFormat = 'csv',
+  extension: GridExportFormatExtension = 'csv',
   filename: string = document.title,
```

Comment:
We could change the default filename to include the date.
something like `export-21.05.2021-19:21.csv`?

Reply:
@dtassone This sounds off-topic to the PR, but we could create a new issue for it 👍